src/samples/msal-b2c-web-sample/src/main/java/com/microsoft/azure/msalwebsample/User.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.msalwebsample;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.json.JSONObject;
+
+@Getter
+@Setter
+public class User {
```

Comment:
Not calling graph, so this can be deleted as well.

Reply:
ok